INSTALL.rst

```diff
@@ -40,7 +47,7 @@ Disable SELinux
 ---------------
 
 The setup described below runs into problems with SELinux related to the sqlite
-database. For now the recommended solution is to simply disable SELinux. In
+database and apache. For now the recommended solution is to simply disable SELinux. In
```

Comment:
If a user is not using SQLite he may skip this step, so wouldn't it be better to mention it (even if we are not sure about the problem)? I think how I edited it may not be the best way. Could you please suggest how to mention this?